src/core/annotation.js

```diff
@@ -95,6 +103,7 @@ class AnnotationFactory {
         attachments,
         xfaDatasets,
         collectFields,
+        needAppearances,
```

Comment:
Given https://github.com/mozilla/pdf.js/blob/987062c302c01bab72645e28a1ffa3fd457e1eb6/src/core/annotation.js#L79 above, could we not simply do something like the following here instead?
```suggestion
        (acroForm && acroForm.get("NeedAppearances)) || false,
```